docs/releases/5.0.txt

```diff
@@ -341,7 +341,8 @@ Signals
 Templates
 ~~~~~~~~~
 
-* ...
+* The new :tfilter:`escapeseq` template filter allows escaping list items, 
+  marking each item in the list as a string that should be auto-escaped.
```

Comment:
```suggestion
* The new :tfilter:`escapeseq` template filter applies :tfilter:`escape` to
  each item in a sequence.
```
A couple of nitpicks 😊
 - Avoid repetition (ie don't say x escapes list items, marking each item to be escaped)
 - avoid specific types (say sequence)
 - auto-escape -> escape? 🤔